src/v0/destinations/klaviyo/transform.js

```diff
@@ -33,6 +33,7 @@ const {
   getSuccessRespEvents,
   checkInvalidRtTfEvents,
   handleRtTfSingleEventError,
+  flattenJson,
```

Comment:
this will flatten the arrays too but Klaviyo supports arrays - no need to flatten an array

Reply:
OS user already handled this via flag in existing flattenJson util - you can refer it, https://github.com/rudderlabs/rudder-transformer/pull/2473/files

Reply:
Done